### ISSUE TITLE
Bug: The app doesn't work on WeChat

### ISSUE THREAD

**Issue by A1eeeee** [FredHappyface/Android.EweSticker#77]
The app doesn't work on WeChat

**Comment by FredHappyface** [comment]
Duplicate - please check for closed issues (eg https://github.com/FredHappyface/Android.EweSticker/issues?q=is%3Aissue+wechat)


https://github.com/FredHappyface/Android.EweSticker/issues/48

**Comment by FredHappyface** [comment]
Can you provide more context? Screenshots would be great if possible

**Comment by A1eeeee** [comment]
> Can you provide more context? Screenshots would be great if possible您能提供更多背景信息吗？如果可能的话，屏幕截图会很棒

It can be used normally in the SMS app. But when using WeChat, there is no response when clicking on the picture in the input method.

**Comment by FredHappyface** [comment]
Yeah different apps may support different input methods. EweSticker works by essentially sending the sticker via an image via the input text, this works fine for the vast majority of tested apps such as WhatsApp, Google messages, discord, telegram etc. Unfortunately I do not have WeChat so cannot test this. Do you get any error codes? From what I can gather I think this is an issue with WeChat rather than this app. Do other sticker keyboards work?

**Comment by A1eeeee** [comment]
> Yeah different apps may support different input methods. EweSticker works by essentially sending the sticker via an image via the input text, this works fine for the vast majority of tested apps such as WhatsApp, Google messages, discord, telegram etc. Unfortunately I do not have WeChat so cannot test this. Do you get any error codes? From what I can gather I think this is an issue with WeChat rather than this app. Do other sticker keyboards work?是的，不同的应用程序可能支持不同的输入法。EweSticker 的工作原理基本上是通过输入文本通过图像发送贴纸，这适用于绝大多数经过测试的应用程序，例如 WhatsApp、Google 消息、discord、telegram 等。不幸的是，我没有微信，所以无法测试这个。您是否收到任何错误代码？从我收集到的信息来看，我认为这是微信的问题，而不是这个应用程序的问题。其他贴纸键盘有用吗？

There aren't any error codes, maybe it's an issue with WeChat

**Comment by A1eeeee** [comment]
> Yeah different apps may support different input methods. EweSticker works by essentially sending the sticker via an image via the input text, this works fine for the vast majority of tested apps such as WhatsApp, Google messages, discord, telegram etc. Unfortunately I do not have WeChat so cannot test this. Do you get any error codes? From what I can gather I think this is an issue with WeChat rather than this app. Do other sticker keyboards work?

There aren't any error codes, maybe it's an issue with WeChat

**Comment by FredHappyface** [comment]
Hi I've updated the share sheet in the latest version. You may need to test with the "enable PNG sticker fallback if sticker format isn't supported" disabled 

For now I'm going to close this ticket, though don't hesitate to follow up if this doesn't solve the issue
